.github/workflows/autobuild.yml

```diff
@@ -1,110 +1,124 @@
 #### Automatically build and upload releases to GitHub ####
 
 on:
+  workflow_dispatch:
   push:
     tags:
-      - "r*" # run this action if a tag beginning with r is created
-  workflow_dispatch:
-
+      - "r*"
```

Comment:
```suggestion
      - "r*"
      - "latest"
```

Reply:
added latest as trigger
decide yourself if you want to use the "latest" tag or leave that code unused/triggered